core/local/local.go

```diff
@@ -224,6 +224,30 @@ func (e *ExecutionScheduler) initVUsConcurrently(
 func (e *ExecutionScheduler) Init(ctx context.Context, samplesOut chan<- stats.SampleContainer) error {
 	logger := e.logger.WithField("phase", "local-execution-scheduler-init")
 
+	// Initialize each executor and do some basic validation.
+	e.state.SetExecutionStatus(lib.ExecutionStatusInitExecutors)
+	logger.Debugf("Start initializing executors...")
+	errMsg := "error while initializing executor %s: %s"
+	exports := e.runner.GetExports()
+	for _, exec := range e.executors {
+		executorConfig := exec.GetConfig()
+		execFn := executorConfig.GetExec().ValueOrZero()
+		execName := executorConfig.GetName()
+
+		if execFn == "" {
+			execFn = "default"
```

Comment:
1. Should we have a constant for this?
2. I don't like having this fallback here, nor having it repeated in `VU.Activate()`. Could this be done better somehow, in the config maybe?